models/image_classification_mobilenet/mobilenet.cpp

```diff
@@ -0,0 +1,140 @@
+#include <fstream>
+#include <iostream>
+#include <sstream>
+
+#include <opencv2/imgproc.hpp>
+#include <opencv2/highgui.hpp>
+#include <opencv2/dnn.hpp>
+#include <opencv2/core/utils/logger.hpp>
+
+using namespace cv;
+using namespace dnn;
+
+std::string keys =
+"{ help  h          | | Print help message. }"
+"{ model m             | image_classification_mobilenetv1_2022apr.onnx | An optional path to file with preprocessing parameters }"
+"{ classes             | labels_imagenet_1k.txt |path to labels file  }"
+"{ input i          | | Path to input image or video file. Skip this argument to capture frames from a camera.}"
+"{ initial_width    | 0 | Preprocess input image by initial resizing to a specific width.}"
+"{ initial_height   | 0 | Preprocess input image by initial resizing to a specific height.}"
+"{ std              | 1.0 1.0 1.0 | Preprocess input image by dividing on a standard deviation.}"
+"{ rgb              | true | swap R and B plane.}"
+"{ crop             | false | Preprocess input image by center cropping.}"
+"{ backend          | 0 | Choose one of computation backends: "
+"0: automatically (by default), "
+"1: Halide language (http://halide-lang.org/), "
+"2: Intel's Deep Learning Inference Engine (https://software.intel.com/openvino-toolkit), "
+"3: OpenCV implementation, "
+"4: VKCOM, "
+"5: CUDA, "
+"6: WebNN }"
+"{ target           | 0 | Choose one of target computation devices: "
+"0: CPU target (by default), "
+"1: OpenCL, "
+"2: OpenCL fp16 (half-float precision), "
+"3: VPU, "
+"4: Vulkan, "
+"6: CUDA, "
+"7: CUDA fp16 (half-float preprocess) }";
```

Comment:
Can you align options with the Python demo?

https://github.com/opencv/opencv_zoo/blob/e6ad1212fb396ceba06f66fcf866536307a84247/models/image_classification_mobilenet/demo.py#L21-L34